module-4/backend/week_three.md

```diff
@@ -1,19 +1,40 @@
 ## Week Three - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. At a high level, what is Node?
+  Node is a runtime for JavaScript that borrows the V8 engine from Chrome to allow JS to function and maintain it's asynchronous, pseudo-multithreaded behavior in a backend context.
 2. What is Express? What is Express similar to in the Ruby world?
+  Express is a backend framework for web applications. It resembles Rails in that it provides an apparatus for interfacing with a database and receiving HTTP requests.
```

Comment:
Yes! It's actually probably more similar to Sinatra as it is lightweight and less opinionated than Rails.